src/bin/identities/CommandSearch.ts

```diff
@@ -54,15 +54,21 @@ class CommandSearch extends CommandPolykey {
             pkClient.grpcClient.identitiesInfoGet(providerMessage, auth),
           meta,
         );
+        let output;
+        if (res.getIdentityId()) {
+          output = [
+            parsers.formatIdentityString(
```

Comment:
Can you remove this function and replace it with the actual implementation:

```
`${providerId}:${identityId}`
```

This is a premature abstraction.

These places need to be changed:

```
»» ~/Projects/js-polykey/src
 ♖ ag 'formatIdentityString'              (qa-testing) pts/2 10:28:26
bin/identities/CommandGet.ts
92:              parsers.formatIdentityString(

bin/identities/CommandSearch.ts
61:              parsers.formatIdentityString(

bin/identities/CommandList.ts
101:                parsers.formatIdentityString(

bin/utils/parsers.ts
59:function formatIdentityString(
65:export { parseNumber, parseSecretPath, parseGestaltId, formatIdentityString };
```

Reply:
Done